### PR TITLE
chore: rename passkey

### PR DESCRIPTION
**Pull request by yauheni-deriv**
## Changes:

Please provide a summary of the change.

### Screenshots:

Please provide some screenshots of the change.

### CODE REVIEW

**Review by suisin-deriv — APPROVED** [review]

File: packages/account/src/Sections/Security/Passkeys/components/passkey-card.tsx [hunk 1]
```diff
@@ -40,6 +42,28 @@ export const PasskeyCard = ({ name, last_used, stored_on, icon }: TPasskeyCard)
...
+                    {
+                        text: localize(''),
+                        value: '',
+                        disabled: true,
+                    },
+                    // TODO: remove empty option when 'revoke' is implemented. Empty option is needed for proper working dropdown
+                    // {
+                    //     text: localize('Revoke'),
+                    //     value: 'revoke',
+                    // },
```
Comment (amina-deriv) [inline packages/account/src/Sections/Security/Passkeys/components/passkey-card.tsx:61]: how does UI looks like with empty option

File: packages/account/src/Sections/Security/Passkeys/passkeys.tsx [hunk 2]
```diff
@@ -103,20 +145,16 @@ const Passkeys = observer(() => {
...
         if (passkey_status === PASSKEY_STATUS_CODES.CREATED) {
             passkeysMenuActionEventTrack('create_passkey_continue_trading');
             history.push(routes.traders_hub);
         }
-
         if (passkey_status === PASSKEY_STATUS_CODES.RENAMING) {
-            // TODO: call API to rename passkey
-            // TODO: track event when renaming is done
+            renamePasskey(current_managed_passkey.id, passkey_data?.name ?? current_managed_passkey.name);
+            passkeysMenuActionEventTrack('passkey_rename_success');
```
Comment (amina-deriv) [inline packages/account/src/Sections/Security/Passkeys/passkeys.tsx:154]: should'nt we check if renaming is success before sending "passkey_rename_success"

Comment (yauheni-deriv) [reply packages/account/src/Sections/Security/Passkeys/passkeys.tsx:154]: yes, will move this call 👍

File: packages/account/src/Sections/Security/Passkeys/components/passkey-card.tsx (hunk 1 above)
Comment (yauheni-deriv) [reply packages/account/src/Sections/Security/Passkeys/components/passkey-card.tsx:61]: Screenshot 2024-05-27 at 11 34 33
I'll also clarify this with the rest of the team

**Review by amina-deriv — APPROVED** [review]

**Pull request merged**